### ISSUE TITLE
Packer fails on eu-central-1 region

### ISSUE THREAD

**Issue by igmar** [mitchellh/packer#1646]
Packer tools fail on the eu-central-1 region. The same happens with aws-cli < 1.5.4, because on that region, the v4 signature is mandatory.

It ends up with a : 
amazon-ebs: Error querying AMI: AWS was not able to validate the provided access credentials (AuthFailure)

The access keys are correct: The same credentials work fine with aws-cli and any region other then eu-central-1.

**Comment by nerd0** [comment]
:+1: Same issue here, is this issue actively being looked at?

**Comment by januzis** [comment]
Same here with `0.7.5`

**Comment by nchammas** [comment]
Same here with 0.7.5. Packer fails when trying to copy a built AMI from `us-east-1` to `eu-central-1`.

The error output is (formatted slightly for readability):

```
1422134478,,ui,error,Build 'spark:base:us-east-1:ebs:pv' errored: 1 error(s) occurred:
* Error Copying AMI (ami-747f051c) to region ({
eu-central-1 
https://ec2.eu-central-1.amazonaws.com
https://s3-eu-central-1.amazonaws.com  %!s(bool=true) %!s(bool=true)  
https://sns.eu-central-1.amazonaws.com
https://sqs.eu-central-1.amazonaws.com 
https://iam.amazonaws.com
https://elasticloadbalancing.eu-central-1.amazonaws.com
https://autoscaling.eu-central-1.amazonaws.com
https://rds.eu-central-1.amazonaws.com
https://route53.amazonaws.com
}):
AWS was not able to validate the provided access credentials (AuthFailure)
```

Packer has no trouble copying this AMI to any other region except `cn-north-1`.

**Comment by Mil0dV** [comment]
As a workaround it's possible to use the aws cli to copy an ami to another region: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/CopyingAMIs.html

**Comment by lutnos** [comment]
If copying, note that permissions and tags are not copied (at least by the AWS console), so will have to be duplicated onto the image after copying.

Also watch out, if trying to copy using packer and including eu-central-1. After the failure copy images will have been created in other regions, but the tags and permissions will not have been set on those copies.

**Comment by femueller** [comment]
+1 same here, using 0.7.5

**Comment by tisba** [comment]
Maybe we are getting lucky and can use packer in every region again soonish: https://github.com/hashicorp/terraform/issues/836#issuecomment-74716205

**Comment by basetta** [comment]
+1 same here, using 0.7.5

**Comment by zaargy** [comment]
+1 using 0.7.5.

**Comment by csabapalfi** [comment]
You can just use [lalyos/packer](https://registry.hub.docker.com/u/lalyos/packer/) via docker. It applies https://github.com/mitchellh/goamz/pull/154 to fix the signature issues.

**Comment by passioncoder** [comment]
+1

**Comment by iangelov** [comment]
I'm subscribing to this as well. Of course, everyone could use a different region or the aforementioned docker container before this is fixed, so at least this isn't that much of a pain.

**Comment by catsby** [comment]
This was fixed by moving to github.com/aws/aws-sdk-go.

Let me know if you're still having issues here

**Comment by tisba** [comment]
@catsby is there an ETA for a release containing the fix?

**Comment by catsby** [comment]
@tisba We're planning on one at the end of the month, but that's the best I can say, sorry

**Comment by ldanielw1** [comment]
+1 - packer integration would be super helpful - please make it happen :)

**Comment by jaytaylor** [comment]
Getting this error w/ 0.8.0 on us-west-2.

**Comment by jaytaylor** [comment]
@catsby I'm still hitting this issue with 0.8.0..

    amazon-ebs output will be in this color.
    
    ==> amazon-ebs: Prevalidating AMI Name...
    ==> amazon-ebs: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
    ==> amazon-ebs:         status code: 401, request id: []
    Build 'amazon-ebs' errored: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
            status code: 401, request id: []

**Comment by mitchellh** [comment]
The credentials may be incorrect. I just verified with both us-west-2 and the eu region locally and it works.

**Comment by jaytaylor** [comment]
Weird, I've verified the credentials work on us-east-1, and still getting
the same error for us-west-2.

**Comment by jaytaylor** [comment]
@mitchellh Is there anything else that could be causing this error?  I wish there was a verbose mode where I could see more detail about what was going on..

**Comment by jaytaylor** [comment]
Update: the above error occurs when running under Linux.

I tried building with the exact same packer.json file under OS-X and things seem to be working fine.

**Comment by catsby** [comment]
@jaytaylor – how are you authenticating? Do you by chance have `~/.aws/credentials` file with different credentials on different machines? 

I attempted to reproduce this on my OS X laptop and inside an Ubuntu VM, and Packer built the image(s) in several regions when using the same ENV vars for authentication.

**Comment by jaytaylor** [comment]
@catsby 

On the failing machines:

They are all Linux.

There are no ~/.aws directories.

There are no *AWS* environment variables.

Auth was specified in the packer .json file:

```json
{
    "builders": [
        {
            "type": "amazon-ebs",
            "access_key": "...",
            "secret_key": "...",
            "region": "us-west-2", ...
```

Without being able to turn on more verbose logging to know what values packer is actually sending it is difficult to reason about this sort of issue.

**Comment by catsby** [comment]
Using the environment variable `PACKER_LOG=1` will output more verbose logging, if that's helpful

**Comment by jaytaylor** [comment]
Very helpful, thank you.

Time permitting I'll make my best effort to follow up on this.

Thank you Clint!

**Comment by catsby** [comment]
Much appreciated @jaytaylor , please report what you find if possible

**Comment by ldanielw1** [comment]
Looks like things are working for me now! Thanks so much!

**Comment by fsouza** [comment]
@jaytaylor did you manage to fix it? Seems like I'm facing the same issue, works like a charm on OS X, fails on Linux, using the same credentials.

What's interesting is that we use it to generate new AMIs nightly, and it started failing last week, but with no upgrades on Packer or changes in credentials :-S I'm using sa-east-1, BTW.

Here's how to reproduce:

```bash
% uname -a
Linux packer-builder 3.19.0-25-generic #26~14.04.1-Ubuntu SMP Fri Jul 24 21:16:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
% git clone https://github.com/tsuru/tsuru-packer.git
% cd tsuru-packer
% packer build -only amazon-ebs tsuru-nightly # should fail, no credentials
amazon-ebs output will be in this color.

==> amazon-ebs: Prevalidating AMI Name...
==> amazon-ebs: Error querying AMI: NoCredentialProviders: no valid providers in chain
Build 'amazon-ebs' errored: Error querying AMI: NoCredentialProviders: no valid providers in chain

==> Some builds didn't complete successfully and had errors:
--> amazon-ebs: Error querying AMI: NoCredentialProviders: no valid providers in chain

==> Builds finished but no artifacts were created.
% export AWS_DEFAULT_REGION=us-east-1 AWS_SECRET_ACCESS_KEY=*** AWS_ACCESS_KEY_ID=*** PACKER_LOG=1
% packer build -only amazon-ebs tsuru-nightly.json
2015/12/01 08:22:15 [INFO] Packer version: 0.8.6  f8f7b7a34c1be06058f5aca23a51247db12cdbc5+CHANGES
2015/12/01 08:22:15 Packer Target OS/Arch: linux amd64
2015/12/01 08:22:15 Built with Go Version: go1.5
2015/12/01 08:22:15 [DEBUG] Discovered plugin: amazon-chroot = /usr/local/bin/packer-builder-amazon-chroot
2015/12/01 08:22:15 [DEBUG] Discovered plugin: amazon-ebs = /usr/local/bin/packer-builder-amazon-ebs
2015/12/01 08:22:15 [DEBUG] Discovered plugin: amazon-instance = /usr/local/bin/packer-builder-amazon-instance
2015/12/01 08:22:15 [DEBUG] Discovered plugin: digitalocean = /usr/local/bin/packer-builder-digitalocean
2015/12/01 08:22:15 [DEBUG] Discovered plugin: docker = /usr/local/bin/packer-builder-docker
2015/12/01 08:22:15 [DEBUG] Discovered plugin: file = /usr/local/bin/packer-builder-file
2015/12/01 08:22:15 [DEBUG] Discovered plugin: googlecompute = /usr/local/bin/packer-builder-googlecompute
2015/12/01 08:22:15 [DEBUG] Discovered plugin: null = /usr/local/bin/packer-builder-null
2015/12/01 08:22:15 [DEBUG] Discovered plugin: openstack = /usr/local/bin/packer-builder-openstack
2015/12/01 08:22:15 [DEBUG] Discovered plugin: parallels-iso = /usr/local/bin/packer-builder-parallels-iso
2015/12/01 08:22:15 [DEBUG] Discovered plugin: parallels-pvm = /usr/local/bin/packer-builder-parallels-pvm
2015/12/01 08:22:15 [DEBUG] Discovered plugin: qemu = /usr/local/bin/packer-builder-qemu
2015/12/01 08:22:15 [DEBUG] Discovered plugin: virtualbox-iso = /usr/local/bin/packer-builder-virtualbox-iso
2015/12/01 08:22:15 [DEBUG] Discovered plugin: virtualbox-ovf = /usr/local/bin/packer-builder-virtualbox-ovf
2015/12/01 08:22:15 [DEBUG] Discovered plugin: vmware-iso = /usr/local/bin/packer-builder-vmware-iso
2015/12/01 08:22:15 [DEBUG] Discovered plugin: vmware-vmx = /usr/local/bin/packer-builder-vmware-vmx
2015/12/01 08:22:15 [DEBUG] Discovered plugin: artifice = /usr/local/bin/packer-post-processor-artifice
2015/12/01 08:22:15 [DEBUG] Discovered plugin: atlas = /usr/local/bin/packer-post-processor-atlas
2015/12/01 08:22:15 [DEBUG] Discovered plugin: compress = /usr/local/bin/packer-post-processor-compress
2015/12/01 08:22:15 [DEBUG] Discovered plugin: docker-import = /usr/local/bin/packer-post-processor-docker-import
2015/12/01 08:22:15 [DEBUG] Discovered plugin: docker-push = /usr/local/bin/packer-post-processor-docker-push
2015/12/01 08:22:15 [DEBUG] Discovered plugin: docker-save = /usr/local/bin/packer-post-processor-docker-save
2015/12/01 08:22:15 [DEBUG] Discovered plugin: docker-tag = /usr/local/bin/packer-post-processor-docker-tag
2015/12/01 08:22:15 [DEBUG] Discovered plugin: vagrant = /usr/local/bin/packer-post-processor-vagrant
2015/12/01 08:22:15 [DEBUG] Discovered plugin: vagrant-cloud = /usr/local/bin/packer-post-processor-vagrant-cloud
2015/12/01 08:22:15 [DEBUG] Discovered plugin: vsphere = /usr/local/bin/packer-post-processor-vsphere
2015/12/01 08:22:15 [DEBUG] Discovered plugin: ansible-local = /usr/local/bin/packer-provisioner-ansible-local
2015/12/01 08:22:15 [DEBUG] Discovered plugin: chef-client = /usr/local/bin/packer-provisioner-chef-client
2015/12/01 08:22:15 [DEBUG] Discovered plugin: chef-solo = /usr/local/bin/packer-provisioner-chef-solo
2015/12/01 08:22:15 [DEBUG] Discovered plugin: file = /usr/local/bin/packer-provisioner-file
2015/12/01 08:22:15 [DEBUG] Discovered plugin: powershell = /usr/local/bin/packer-provisioner-powershell
2015/12/01 08:22:15 [DEBUG] Discovered plugin: puppet-masterless = /usr/local/bin/packer-provisioner-puppet-masterless
2015/12/01 08:22:15 [DEBUG] Discovered plugin: puppet-server = /usr/local/bin/packer-provisioner-puppet-server
2015/12/01 08:22:15 [DEBUG] Discovered plugin: salt-masterless = /usr/local/bin/packer-provisioner-salt-masterless
2015/12/01 08:22:15 [DEBUG] Discovered plugin: shell = /usr/local/bin/packer-provisioner-shell
2015/12/01 08:22:15 [DEBUG] Discovered plugin: shell-local = /usr/local/bin/packer-provisioner-shell-local
2015/12/01 08:22:15 [DEBUG] Discovered plugin: windows-restart = /usr/local/bin/packer-provisioner-windows-restart
2015/12/01 08:22:15 [DEBUG] Discovered plugin: windows-shell = /usr/local/bin/packer-provisioner-windows-shell
2015/12/01 08:22:15 Detected home directory from env var: /home/ubuntu
2015/12/01 08:22:15 Detected home directory from env var: /home/ubuntu
2015/12/01 08:22:15 Attempting to open config file: /home/ubuntu/.packerconfig
2015/12/01 08:22:15 [WARN] Config file doesn't exist: /home/ubuntu/.packerconfig
2015/12/01 08:22:15 Packer config: &{DisableCheckpoint:false DisableCheckpointSignature:false PluginMinPort:10000 PluginMaxPort:25000 Builders:map[amazon-instance:/usr/local/bin/packer-builder-amazon-instance null:/usr/local/bin/packer-builder-null parallels-iso:/usr/local/bin/packer-builder-parallels-iso qemu:/usr/local/bin/packer-builder-qemu virtualbox-iso:/usr/local/bin/packer-builder-virtualbox-iso virtualbox-ovf:/usr/local/bin/packer-builder-virtualbox-ovf vmware-iso:/usr/local/bin/packer-builder-vmware-iso amazon-chroot:/usr/local/bin/packer-builder-amazon-chroot amazon-ebs:/usr/local/bin/packer-builder-amazon-ebs digitalocean:/usr/local/bin/packer-builder-digitalocean docker:/usr/local/bin/packer-builder-docker file:/usr/local/bin/packer-builder-file googlecompute:/usr/local/bin/packer-builder-googlecompute openstack:/usr/local/bin/packer-builder-openstack parallels-pvm:/usr/local/bin/packer-builder-parallels-pvm vmware-vmx:/usr/local/bin/packer-builder-vmware-vmx] PostProcessors:map[artifice:/usr/local/bin/packer-post-processor-artifice atlas:/usr/local/bin/packer-post-processor-atlas compress:/usr/local/bin/packer-post-processor-compress docker-import:/usr/local/bin/packer-post-processor-docker-import docker-tag:/usr/local/bin/packer-post-processor-docker-tag vagrant:/usr/local/bin/packer-post-processor-vagrant vagrant-cloud:/usr/local/bin/packer-post-processor-vagrant-cloud docker-push:/usr/local/bin/packer-post-processor-docker-push docker-save:/usr/local/bin/packer-post-processor-docker-save vsphere:/usr/local/bin/packer-post-processor-vsphere] Provisioners:map[chef-client:/usr/local/bin/packer-provisioner-chef-client chef-solo:/usr/local/bin/packer-provisioner-chef-solo file:/usr/local/bin/packer-provisioner-file puppet-masterless:/usr/local/bin/packer-provisioner-puppet-masterless salt-masterless:/usr/local/bin/packer-provisioner-salt-masterless shell:/usr/local/bin/packer-provisioner-shell shell-local:/usr/local/bin/packer-provisioner-shell-local windows-restart:/usr/local/bin/packer-provisioner-windows-restart ansible-local:/usr/local/bin/packer-provisioner-ansible-local powershell:/usr/local/bin/packer-provisioner-powershell puppet-server:/usr/local/bin/packer-provisioner-puppet-server windows-shell:/usr/local/bin/packer-provisioner-windows-shell]}
2015/12/01 08:22:15 Setting cache directory: /home/ubuntu/tsuru-packer/packer_cache
2015/12/01 08:22:15 Detected home directory from env var: /home/ubuntu
2015/12/01 08:22:15 Loading builder: amazon-ebs
2015/12/01 08:22:15 Creating plugin client for path: /usr/local/bin/packer-builder-amazon-ebs
2015/12/01 08:22:15 Starting plugin: /usr/local/bin/packer-builder-amazon-ebs []string{"/usr/local/bin/packer-builder-amazon-ebs"}
2015/12/01 08:22:15 Waiting for RPC address for: /usr/local/bin/packer-builder-amazon-ebs
2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Plugin minimum port: 10000
2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Plugin maximum port: 25000
2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Plugin address: unix /tmp/packer-plugin916437200
2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Waiting for connection...
2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Serving a plugin connection...
2015/12/01 08:22:15 Loading provisioner: shell
2015/12/01 08:22:15 Creating plugin client for path: /usr/local/bin/packer-provisioner-shell
2015/12/01 08:22:15 Starting plugin: /usr/local/bin/packer-provisioner-shell []string{"/usr/local/bin/packer-provisioner-shell"}
2015/12/01 08:22:15 Waiting for RPC address for: /usr/local/bin/packer-provisioner-shell
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin minimum port: 10000
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin maximum port: 25000
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin address: unix /tmp/packer-plugin228148138
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Waiting for connection...
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Serving a plugin connection...
2015/12/01 08:22:15 Loading provisioner: shell
2015/12/01 08:22:15 Creating plugin client for path: /usr/local/bin/packer-provisioner-shell
2015/12/01 08:22:15 Starting plugin: /usr/local/bin/packer-provisioner-shell []string{"/usr/local/bin/packer-provisioner-shell"}
2015/12/01 08:22:15 Waiting for RPC address for: /usr/local/bin/packer-provisioner-shell
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin minimum port: 10000
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin maximum port: 25000
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Plugin address: unix /tmp/packer-plugin115988822
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Waiting for connection...
2015/12/01 08:22:15 packer-provisioner-shell: 2015/12/01 08:22:15 Serving a plugin connection...
2015/12/01 08:22:15 Loading post-processor: vagrant
2015/12/01 08:22:15 Creating plugin client for path: /usr/local/bin/packer-post-processor-vagrant
2015/12/01 08:22:15 Starting plugin: /usr/local/bin/packer-post-processor-vagrant []string{"/usr/local/bin/packer-post-processor-vagrant"}
2015/12/01 08:22:15 Waiting for RPC address for: /usr/local/bin/packer-post-processor-vagrant
2015/12/01 08:22:15 packer-post-processor-vagrant: 2015/12/01 08:22:15 Plugin minimum port: 10000
2015/12/01 08:22:15 packer-post-processor-vagrant: 2015/12/01 08:22:15 Plugin maximum port: 25000
2015/12/01 08:22:15 packer-post-processor-vagrant: 2015/12/01 08:22:15 Plugin address: unix /tmp/packer-plugin935782500
2015/12/01 08:22:15 packer-post-processor-vagrant: 2015/12/01 08:22:15 Waiting for connection...
2015/12/01 08:22:15 packer-post-processor-vagrant: 2015/12/01 08:22:15 Serving a plugin connection...
2015/12/01 08:22:15 ui: amazon-ebs output will be in this color.
2015/12/01 08:22:15 ui:
2015/12/01 08:22:15 Build debug mode: false
2015/12/01 08:22:15 Force build: false
2015/12/01 08:22:15 Preparing build: amazon-ebs
amazon-ebs output will be in this color.

2015/12/01 08:22:15 packer-builder-amazon-ebs: 2015/12/01 08:22:15 Config: {PackerConfig:{PackerBuildName:amazon-ebs PackerBuilderType:amazon-ebs PackerDebug:false PackerForce:false PackerUserVars:map[aws_access_key:<Filtered> aws_secret_key:<Filtered>]} AccessConfig:{AccessKey:<Filtered> SecretKey:<Filtered> RawRegion:us-east-1 Token:} AMIConfig:{AMIName:tsuru-nightly 1448965335 AMIDescription: AMIVirtType: AMIUsers:[] AMIGroups:[] AMIProductCodes:[] AMIRegions:[] AMITags:map[] AMIEnhancedNetworking:false AMIForceDeregister:false} BlockDevices:{AMIMappings:[] LaunchMappings:[]} RunConfig:{AssociatePublicIpAddress:false AvailabilityZone: IamInstanceProfile: InstanceType:m3.medium RunTags:map[] SourceAmi:ami-d05e75b8 SpotPrice: SpotPriceAutoProduct: SecurityGroupId: SecurityGroupIds:[sg-3df7a255] SubnetId: TemporaryKeyPairName:packer 565d74d7-66e8-5d32-07f7-f7298ff8556a UserData: UserDataFile: WindowsPasswordTimeout:10m0s VpcId: Comm:{Type:ssh SSHHost: SSHPort:22 SSHUsername:ubuntu SSHPassword: SSHPrivateKey: SSHPty:false SSHTimeout:5m0s SSHDisableAgent:false SSHHandshakeAttempts:10 SSHBastionHost: SSHBastionPort:0 SSHBastionUsername: SSHBastionPassword: SSHBastionPrivateKey: WinRMUser: WinRMPassword: WinRMHost: WinRMPort:0 WinRMTimeout:0} SSHKeyPairName: SSHPrivateIp:false} ctx:{Data:<nil> Funcs:map[clean_ami_name:0x51ee50] UserVariables:map[aws_access_key:<Filtered> aws_secret_key:<Filtered>] EnableEnv:false BuildName:amazon-ebs BuildType:amazon-ebs TemplatePath:/home/ubuntu/tsuru-packer/tsuru-nightly.json}}
2015/12/01 08:22:15 Waiting on builds to complete...
2015/12/01 08:22:15 Starting build run: amazon-ebs
2015/12/01 08:22:15 Running builder: amazon-ebs
2015/12/01 08:22:15 ui: ==> amazon-ebs: Prevalidating AMI Name...
==> amazon-ebs: Prevalidating AMI Name...
2015/12/01 08:22:16 ui error: ==> amazon-ebs: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
==> amazon-ebs: 	status code: 401, request id: []
==> amazon-ebs: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
2015/12/01 08:22:16 ui error: Build 'amazon-ebs' errored: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: []
==> amazon-ebs: 	status code: 401, request id: []
2015/12/01 08:22:16 Builds completed. Waiting on interrupt barrier...
2015/12/01 08:22:16 machine readable: error-count []string{"1"}
2015/12/01 08:22:16 ui error:
==> Some builds didn't complete successfully and had errors:
2015/12/01 08:22:16 machine readable: amazon-ebs,error []string{"Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials\n\tstatus code: 401, request id: []"}
2015/12/01 08:22:16 ui error: --> amazon-ebs: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: []
2015/12/01 08:22:16 ui:
==> Builds finished but no artifacts were created.
2015/12/01 08:22:16 waiting for all plugin processes to complete...
Build 'amazon-ebs' errored: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: []

==> Some builds didn't complete successfully and had errors:
2015/12/01 08:22:16 /usr/local/bin/packer-post-processor-vagrant: plugin process exited
2015/12/01 08:22:16 /usr/local/bin/packer-provisioner-shell: plugin process exited
--> amazon-ebs: Error querying AMI: AuthFailure: AWS was not able to validate the provided access credentials
	status code: 401, request id: []

==> Builds finished but no artifacts were created.
2015/12/01 08:22:16 /usr/local/bin/packer-builder-amazon-ebs: plugin process exited
2015/12/01 08:22:16 /usr/local/bin/packer-provisioner-shell: plugin process exited
```

Running the exact set of commands on OS X works as expected (including the error message when the environment variables are not defined). Both boxes are running packer 0.8.6.

Should I open a new issue?
